### DISCUSSION TITLE
Issues with latest docker image

### DISCUSSION THREAD

**Discussion by alicemara** [jertel/elastalert2#1265] [Q&A] [answered]
I don't necessarily need help but wanted to share some issues I ran into with the latest docker image (says it's from 9/09/23).

Trying to deploy a container with the latest image resulted with the container immediately failing with this error:  
```
Traceback (most recent call last):
  File "/usr/local/bin/elastalert", line 5, in <module>
    from elastalert.elastalert import main
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 32, in <module>
    from elastalert.alerters.debug import DebugAlerter
  File "/usr/local/lib/python3.11/site-packages/elastalert/alerters/debug.py", line 1, in <module>
    from elastalert.alerts import Alerter, BasicMatchString
  File "/usr/local/lib/python3.11/site-packages/elastalert/alerts.py", line 8, in <module>
    from prettytable import PrettyTable
ModuleNotFoundError: No module named 'prettytable'
```  
I was able to replicate the issue on multiple setups that have Elasticsearch 8.1/8.9 including a setup where I replaced the Elastalert2 container running image 2.13.2 with latest and it gave the same error above, and immediately started working again when I reverted to 2.13.2.

I highly appreciate this project and everyone who maintains it. Please let me know if there's any further info I can give to help!

**Comment by jertel** [comment, marked as answer]
Thank you for raising this issue! I reproduced it and pushed a fix to ensure the missing package is included in the image. Please let me know if you continue seeing an issue.

**Reply by alicemara** [reply]
Initial testing seems to show it's fixed! Thank you for such a quick fix!
